.github/workflows/build.yml

```diff
@@ -0,0 +1,96 @@
+on:
+  push:
+    branches:
+      - "graal-build-v2"
+jobs:
+  build-jar:
+    runs-on: "ubuntu-20.04"
+    steps:
+      - uses: "actions/checkout@v2"
+      - uses: "actions/cache@v2"
+        with:
+          path: |
+            ~/.m2/repository
+            ~/.gitlibs
+          key: "${{ runner.os }}-deps-${{ hashFiles('deps.edn') }}"
+      - uses: "DeLaGuardo/setup-clojure@4.0"
+        with:
+          cli: "1.10.3.1075"
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+      - run: "clojure -A:dev -M --report stderr -m moclojer.build"
+        env:
+          CI_COMMIT_SHA: "${{ github.sha }}"
+      - uses: "actions/upload-artifact@v2"
+        with:
+          name: "moclojer-${{ github.run_id }}-jar"
+          path: |
+            target/*.jar
+            target/native
+  build-native:
+    needs: "build-jar"
+    runs-on: "${{ matrix.os }}"
+    name: "Build native image on ${{ matrix.os }}"
+    strategy:
+      matrix:
+        os: [ "macos-11", "ubuntu-20.04" ]
+    steps:
+      - uses: "graalvm/setup-graalvm@v1"
+        with:
+          version: "22.0.0.2"
+          java-version: "17"
+          components: "native-image"
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+      - uses: "actions/checkout@v2"
+      - uses: "actions/download-artifact@v2"
+        with:
+          name: "moclojer-${{ github.run_id }}-jar"
+          path: "target"
+      - name: "Gen reflect.config.json"
+        working-directory: "target/native"
+        run: |
+          echo [] > moclojer.yml
```

Comment:
To address #6, we need to parse a YAML file during this reflect.config process.

Once we parse a YAML on this stage, the YAML library will be added to the reflect config.